src/test/java/com/netflix/imflibrary/app/IMPAnalyzerTestApp2E2022.java

```diff
@@ -0,0 +1,58 @@
+package com.netflix.imflibrary.app;
+
+import com.netflix.imflibrary.IMFErrorLogger;
+import com.netflix.imflibrary.IMFErrorLoggerImpl;
+import com.netflix.imflibrary.st2067_2.ApplicationCompositionFactory;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import testUtils.TestHelper;
+
+import java.io.File;
+import java.io.IOException;
+
+@Test(groups = "unit")
+public class IMPAnalyzerTestApp2E2022
```

Comment:
Please change class, filename and test content folder to reflect "App2E**2021**" to avoid confusion.